### PR TITLE
Default currency for additional locales

### PR DESCRIPTION
**Pull request by TheoChevalier**
@ScottDowne added currencies for Argentina & Chile (Stripe only) + assigned a default currency for several locales. Does that look good?
I’m not sure how to test the localeCountryData array, however. What is it used for?

Also, alpha sorted the arrays for improved readability

### CODE REVIEW

**Comment by ScottDowne** [comment]
I'm pretty sure localeCountryData is dead code. We used to have a form that had a country dropdown, and it would be pre selected based on locale.

I havn't had the time to confirm and remove it, but I was planning on doing that soon.

File: src/lib/get-messages.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import assign from 'object-assign';
 import locales from '../../public/locales.json';
 
 function getMessages(locale) {
-  if (locale === 'es-MX') {
+  if (['es-AR', 'es-CL', 'es-MX'].indexOf(locale) >= 0) {
```
Comment (ScottDowne) [inline src/lib/get-messages.js]: Should we add es-AR and es-CL empty locale folders with a `.keep` file to the repo to expose these locales?

**Comment by ScottDowne** [comment]
Looks pretty good to me. One comment about a thing I think we can easily add.

File: src/lib/get-messages.js (hunk 1 above)
Comment (TheoChevalier) [inline src/lib/get-messages.js]: Yes, I actually did it, but forgot to git add :(

**Comment by TheoChevalier** [comment]
Should be better, good catch :)

**Pull request merged**